CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ SET(BEMENU_DESCRIPTION "Dynamic menu library and client program inspired by dmen
 SET(BEMENU_VERSION "0.1.0")
 SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/CMake)
 
+OPTION(BEMENU_CURSES_RENDERER "Build curses backend" ON)
+OPTION(BEMENU_X11_RENDERER "Build X11 backend" ON)
+OPTION(BEMENU_WAYLAND_RENDERER "Build wayland renderer" OFF)
```

Comment:
Instead of options, could also use single variable to customize the renderers, `BEMENU_RENDERERS` or something with default value `"curses;x11"` it could then be modified from command line with `-DBEMENU_RENDERERS="curses;x11;wayland"` for example.

Reply:
I don't like this because of how source-based distro packaging works; it is easier to assign a variable to an option/USE instead of using a list.

In exherbo packages format (called exheres) it would as simple as:

```bash
CMAKE_SRC_CONFIGURE_OPTIONS=(
    'ncurses BEMENU_CURSES_RENDERER'
    'X BEMENU_X11_RENDERER'
    'wayland BEMENU_WAYLAND_RENDERER'
)
```

While a `BEMENU_RENDERERS` would require creating a list in bash and appending the renderers.

Reply:
Fair enough.